changelog.txt

```diff
@@ -0,0 +1,2 @@
+fixed - fixes bug in Cloud Firestore emulator where rulesets with no-op writes would modify a document's updated_at timestamp
+changed - reduced lock contention in Cloud Firestore emulator during concurrent writes to a document
```

Comment:
in practice this completely eliminates the issues reported in https://github.com/firebase/quickstart-nodejs/issues/64